packages/react-scripts/scripts/init.js

```diff
@@ -135,6 +135,21 @@ module.exports = function(
     return;
   }
 
+  // modifies README.md commands based on user used package manager.
+  if (useYarn) {
+    try {
+      const data = fs.readFileSync(path.join(appPath, 'README.md'), 'utf8');
```

Comment:
Rename `data` to `readme` and modify that directly instead of creating another `formatted` variable.

Reply:
@iansu I renamed variable to the readme and modifying directly. I used variable naming reference from the same file.